### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by goldenuni**

### CODE REVIEW

File: app/cinema/hall.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str, customers: [Customer],
+                      cleaning_staff: Cleaner) -> None:
```
Comment (olena-hul) [inline app/cinema/hall.py:11]: Place each argument in a separate line and annotation for customers is wrong

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+from app.cinema.bar import CinemaBar
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list, hall_number: int,
+                 cleaner: str, movie: str) -> None:
```
Comment (olena-hul) [inline app/main.py:8]: Place each argument in a separate line

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+    cb = CinemaBar()
+
+    cinema_customers = []
+    for customer in customers:
+        cinema_customers.append(Customer(customer["name"], customer["food"]))
```
Comment (olena-hul) [inline app/main.py:13]: Use list comprehension

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
v1

**Review by Bohdan-Salamakha — APPROVED** [review]

**Pull request closed without merging**